Library/Formula/postgresql.rb

```diff
@@ -45,6 +45,10 @@ class Postgresql < Formula
   def install
     ENV.libxml2 if MacOS.version >= :snow_leopard
 
+    ENV.prepend "LDFLAGS", "-L#{Formula['openssl'].opt_lib}"
+    ENV.prepend "CFLAGS", "-L#{Formula['openssl'].opt_lib}"
```

Comment:
Yeah, include was intended.
Actually CPPFLAGS is enough, will push version without CFLAGS modification.